help/en/html/tools/speedometer/index.shtml

```diff
@@ -13,10 +13,123 @@
+      <ul>
+      <li>If you select "entry" for all sensors, so that timing starts and stops when a train enters a block, 
+      then enter the distance from the start of block 1 to the start of block 2 (<strong>Distance 1</strong>), 
+      and the distance from the start of block 1 to the start of block 3 (<strong>Distance 2</strong>)).</li>
+      <li>If you select "exit" for all sensors, so that timing will start or stop when a train exits a block, 
+      then enter the distance from the end of block 1 to the end of block 2 (<strong>Distance 1</strong>), and
+      the distance from the end of block 1 to the end of block 3 (<strong>Distance 2</strong>) as the two distances.</li>
+      <li>If you select "entry" for any sensor and "exit" for another, then enter the distance from the start of
+      any block where you selected "entry" to the end of any block where you selected "exit" as your distances.</li>
+      </ul>
+
+
+
+      <!-- ************************* ****************************** **************************** ************* -->
+      
+      <h2 id="location">Using location sensors</h2>
+      
+      <p>If your railroad has two or more "spot" or location sensors, e.g. IR or photoresistors that record a train 
+      passing by the sensor rather than occupying a block of track, you can similarly use these sensors to trigger 
+      speedometer measurement.</p>
+      
+      <p>Enter the system names for the sensors you wish to use for speed measurement.  </p>
+      
+      <p>The distances you enter in fields <strong>Distance 1</strong> and <strong>Distance 2</strong> should be
+      measured according to whether you select <strong>"entry"</strong> or <strong>"exit"</strong> for any
+      paticular location sensor:</p>
+      
+      <ul>
+      <li>You should select either "entry" or "exit" for all sensors, so that timing starts and stops when a train
+      first enters the field of view of a sensor, or when it leaves the field of view.</li>  
+      <li>Enter the distance from sensor 1 to sensor 2 (<strong>Distance 1</strong>), and the distance from sensor
+      1 to sensor 3 (<strong>Distance 2</strong>) as the two distances.</li>
+      <li>Do NOT select "entry" for some sensors and "exit" for others as this will result in erroneous speed
+      calculations.</li>
+      </ul>
+
+      
+      <!-- ************************* ****************************** **************************** ************* -->
+      
+      <h2 id="single">Using a single block occupancy sensor</h2>
+
+      <p>You can also set up the Speedometer to measure speed across a single block. In this case, the same 
+      block occupancy sensor name should be entered for both the first sensor and the second sensor. The first sensor 
+      should be set to <strong>"entry"</strong> and the second sensor should be set to <strong>"exit"</strong>.  
+      <strong>"Distance 1"</strong> should be set to the length of the block.</p>
```

Comment:
It should actually be the length of the block plus the length of the detected locomotive/train, as it has to get all the way out.  This also relies on the block sensor having the same delay to go on as to go off; many go off more slowly than on.

One-block operation is really meant for a loop of track, where you can use "entry" for both.

Reply:
Not sure I understand your point about a loop of track: use same sensor for 1 and 2 and "entry" for both?  Will fix text for your other comments as well.  Thanks.

Reply:
It's not a common case, but you can measure speed if you have a loop (perhaps an existing loop on the layout) and just a single block sensor wired up.  You measure from entry to entry on that single block sensor, start of lap to end of lap.  Think of the start of the block as a start/finish line.  By using "entry" for both start and end, you don't have to worry about whether the block detector has equal on and off delays.

Reply:
Thanks.  I think I have it now...